README.md

```diff
@@ -1,8 +1,8 @@
 # Kasa
-English to Twi translation library/system being put together by the Ghana NLP team.
+A python library/system to interact with the African language translation, STT (Speech to Text) and TTS (Text to Speech) models put together by the Ghana NLP team.
```

Comment:
we will have to add more details here. For now the main idea is for the library to expose NLP tools for Low-resourced languages with a focus on African Languages. You can then include some minor details of features we have now, such as a module (khaya) for accessing the APIs.

Reply:
Alright I have taken this into consideration, can I give a good summary of how the NLP tools work and then prolly link up a blog series I am putting together to go into more details on it?

Check it out here
https://pkwolffe.hashnode.dev/series/build-with-ghananlp

Reply:
yeah that sounds like a plan. we can include that in a section "further reading". give it a try and lets review together